src/actions/pubSubHandlers.js

```diff
@@ -165,13 +168,24 @@ export function handleJoinedChannel({ user: userId, channel: channelId }) {
   }
 }
 
+const handleCurrentUserLeftChannel = () => (dispatch, getState) => {
+  const channels = joinedChannelsSelector(getState())
+  if (channels) {
+    dispatch(goToLastUsedChannel())
+  } else {
+    dispatch(setIntialDataLoading(false))
+    dispatch(showSidebar(false))
+    dispatch(goTo('/chat'))
+  }
+}
+
 export function handleLeftChannel({ user: userId, channel: channelId }) {
   return (dispatch, getState) => {
     dispatch({
       type: types.REMOVE_USER_FROM_CHANNEL,
       payload: { channelId, userId },
     })
-
+    dispatch(handleCurrentUserLeftChannel())
     const rooms = joinedRoomsSelector(getState())
```

Comment:
@AleshaOleg this code can never be reached since in every case `handleCurrentUserLeftChannel ` at least will go to `'chat'`. Can you remove it?